docs/serving/autoscaling/scale-to-zero.md

```diff
@@ -7,13 +7,15 @@ type: "docs"
 
 # Configuring scale to zero
 
-**IMPORTANT:** Scale to zero can only be enabled if you are using the Knative Pod Autoscaler (KPA), and can only be configured globally. For more information about using KPA or global configuration, see the documentation on [Autoscaling concepts](./autoscaling-concepts.md).
+!!! warning
+    Scale to zero can only be enabled if you are using the Knative Pod Autoscaler (KPA), and can only be configured globally. For more information about using KPA or global configuration, see the documentation on [Autoscaling concepts](./autoscaling-concepts.md).
```

Comment:
Idk if we want to use KnativePodAutoscaler here as per HoriztonalPodAutoscaler in the k8s style guide https://kubernetes.io/docs/contribute/style/style-guide/#documentation-formatting-standards

cc @knative/serving-reviewers

Reply:
I am down with this